Makefile

```diff
@@ -152,7 +152,18 @@ eunit:
 all-tests: eunit test
 
 system-test:
-	@./rebar3 as system_test do ct --dir system_test --logdir system_test/logs $(CT_TEST_FLAGS)
+	@./rebar3 as system_test do ct --dir system_test --logdir system_test/logs --config system_test/cfg $(CT_TEST_FLAGS)
+
+system-test-deps: | system_test/aest_hard_fork_SUITE_data/db/mnesia_ae-uat-epoch_backup.tar ;
+
+system_test/aest_hard_fork_SUITE_data/db/mnesia_ae-uat-epoch_backup.tar: system_test/aest_hard_fork_SUITE_data/db/mnesia_ae-uat-epoch_backup
+	tar -c -C $(<D) -f $@ $(<F)
+
+system_test/aest_hard_fork_SUITE_data/db/mnesia_ae-uat-epoch_backup: system_test/aest_hard_fork_SUITE_data/db/mnesia_ae-uat-epoch_backup.gz
+	zcat <$< >$@
+
+system_test/aest_hard_fork_SUITE_data/db/mnesia_ae-uat-epoch_backup.gz:
+	curl -fsS --create-dirs -o $@ https://6798-99802036-gh.circle-artifacts.com/0/tmp/chain_snapshots/54.202.11.197/tmp/mnesia_ip-172-31-36-149_db_backup_1523145687.gz
```

Comment:
@acidsennui Here I retrieve the DB snapshot with the chain. It is hardcoded link, I think it is fine for the moment (though I plan to try the newer snapshot on CI in scope of this PR).